imports/stylesheets/incidents/incidentForm.import.styl

```diff
@@ -61,3 +61,10 @@
         margin-bottom -1px
         > a
           color $primary
+
+.suggested
+  .control-label::after
+  label.cumulative::after
+  .tabs .active a::after
+    content " *"
+    color lighten($primary, 50%)
```

Comment:
I'm worried since most users are accustomed to asterisks indicating a field is required there may be some confusion with using them to indicate autofill. Maybe we can change the background color of the `form-group`?
Maybe like so:
<img width="717" alt="screen shot 2017-01-03 at 11 38 36 am" src="https://cloud.githubusercontent.com/assets/4105343/21614913/6ff7a81e-d1a9-11e6-8f06-5be0c17f2450.png">
And we could also add an icon like this: http://fontawesome.io/icon/magic/ with a tooltip explaining it has been autofilled?